constants.gradle

```diff
@@ -1,6 +1,6 @@
 project.ext {
     APPLICATION_ID = "org.edx.mobile"
-    KOTLIN_VERSION = "1.3.41"
+    KOTLIN_VERSION = "1.3.50"
```

Comment:
Any specific reason to update the `Kotlin` version here?

Reply:
This commit explains the issue: https://github.com/edx/edx-app-android/pull/1579/commits/8a11a95d199a9972d5131692824d838cdf1e0ff9

which is:

> Our previously used Kotlin version i.e. v1.3.41 was causing a compile time error when using the prodDebug variant. This version bump fixes the issue.

if you change the version back, you'll be able to see the error.